FCWTAPI/FCWTAPI.cs

```diff
@@ -58,6 +57,22 @@ public static float[][] FixOutputArray(float[] array1D, int size, int noctave, i
             }
             return fixedResults;
         }
-        
+        //First element corresponds to the first jagged array dimension, second element corresponds to the second dim
+        public static float[,] ToTwoDArray(float[][] JaggedTwoD)
```

Comment:
Need a way to make sure that any errors caused by exceeding the bounds of any row array is caught and handled. See: https://stackoverflow.com/questions/26291609/converting-jagged-array-to-2d-array-c-sharp

I think it would also be better if the InvalidOperationException would also print the row number at which the error occurred.